index.js

```diff
@@ -32,56 +32,63 @@ const buttonWork = document.querySelector('.button-work');
 
 function newPopUp(a, b, c, d, e, f) {
   const backPop = document.createElement('div');
-  backPop.classList.add('popbackground');
   const popCard = document.createElement('div');
-  popCard.classList.add('popup-card');
   const headerBox = document.createElement('div');
-  headerBox.classList.add('header-box');
   const clonedHeader = popUp.workHeader[a].cloneNode(true);
-  clonedHeader.classList.add('cloned-header');
   const clonedHeader2 = popUp.workHeader[b].cloneNode(true);
-  clonedHeader2.classList.add('cloned-header');
   const Bclose = document.createElement('img');
-  Bclose.src = 'images/bclose.png';
-  Bclose.classList.add('b-close');
   const clonedSubheader = popUp.subHeader[c].cloneNode(true);
-  clonedSubheader.classList.add('cloned-subheader');
   const clonedImage = popUp.image[d].cloneNode(true);
-  clonedImage.classList.add('cloned-img');
   const clonedDesktopImage = popUp.image[e].cloneNode(true);
-  clonedDesktopImage.classList.add('cloned-desk-img');
   const popInfoContainer = document.createElement('div');
   const popBodyContainer = document.createElement('div');
+  const description = document.createElement('p');
+  const clonedTech = popUp.technologies[f].cloneNode(true);
+  const githubTech = document.createElement('li');
```

Comment:
- [ ] Kindly make sure you use an array of objects instead of an object to display your data as specified by the previous code reviewer 🙏 
- [ ] Kindly make sure you populate every property in your object. For example, the `image` property in your object is just a string please make sure you add the `image` instead 🙏

Reply:
Hello @Ugboaja-Uchechi !
First of all, thank you for your review!
Before I continue I would just like to see what you think about the change that I did to see if you would want me to change the code more.
For making my object an array of objects I did this: 
![popup-code](https://user-images.githubusercontent.com/93748135/180490958-e5d7f824-8e7a-432f-bde7-52dfa2722db8.png)

This way, the main object technically has other objects inside, they look empty at the moment, but I have this line of code to fill it:

![popup-selector](https://user-images.githubusercontent.com/93748135/180491133-865e59ee-3954-41b4-9351-ead79172f90e.png)

That way the objects inside the main object have an array of all the elements I will need to display the work section, now I can select the ones I need for each section with the indexes by using this:
![popup-index](https://user-images.githubusercontent.com/93748135/180492817-28b0252a-4a4d-4f44-ac92-84f9add5855d.png)

So the image object should not be empty now.
I did a console log to make sure they were being added:
![popup-group](https://user-images.githubusercontent.com/93748135/180491402-4da5cc0b-0a20-4d21-b110-14ff2c7ad8cf.png)

I decided to do the main object like this because this makes my code easy to adapt, and in case that another work section is added, I would not have to create another object, the elements will be added automatically to the array, and the next section will be displayed correctly without having to add any code at all on the Javascript file!

What do you think about this? Do you think it is correct by having the code like this or do you think it would be best if I changed the object that I made completely? Changing my object that gets autofilled to another object that already has the information would mean changing a lot of my code structure but I want to make sure I have a code you would feel comfortable accepting. So please let me know if this meets your requirements or if you want me to do a more radical change on the objects.
Thank you!